### PR TITLE
Add Sem & Pract 06

### PR DESCRIPTION
**Pull request by Justsvetoslavov**
Initial notes & tasks.

### CODE REVIEW

File: SI/Sem.06/Pract.06/README.md
Comment (GeorgiTerziev02) [inline SI/Sem.06/Pract.06/README.md:11]: IMO This task should be in the last ones

**Review by GeorgiTerziev02 — APPROVED** [review]

File: SI/Sem.06/README.md
Comment (GeorgiTerziev02) [inline SI/Sem.06/README.md:41]: Add their english names

**Comment by Justsvetoslavov** [comment]
@dejansvetoslavov @plamena-ilieva I will wait for at least 1 more review from 1 of you.

If it is approved without any required changes, just merge it.

**Pull request merged**
